.github/actions-scripts/post-comment.js

```diff
@@ -74,6 +79,76 @@ async function createCommentWithMetricsTable(metrics) {
   }
 }
 
+function getMetricValue(key, value) {
+  let returnValue = value;
+  switch (key) {
+    case `reliability_rating`:
+      switch (value + "") {
+        case "1.0":
+          returnValue = "A (0 Bugs)";
+          break;
+        case "2.0":
+          returnValue = "B (at least 1 Minor Bug)";
+          break;
+        case "3.0":
+          returnValue = "C (at least 1 Major Bug)";
+          break;
+        case "4.0":
+          returnValue = "D (at least 1 Critical Bug)";
+          break;
+        case "5.0":
```

Comment:
Use constant vars

Reply:
Focus on code smells, you have a lot of them. Just making some good changes in the code is not enough. And why is this showing up like this.
<img width="587" alt="image" src="https://github.com/KhushPatel018/kdu-coursework/assets/108662490/97d64cce-0866-4ba1-aeac-588c11e095b1">